src/reporter/coverage-report.xsl

```diff
@@ -196,19 +224,16 @@
       <!-- Analyze the entire stylesheet string. For each matching substring, create a map that
          records the kind of match. -->
       <xsl:variable name="regex-groups" as="map(xs:integer, xs:string)+">
-         <xsl:analyze-string select="$stylesheet-string"
-            regex="{$construct-regex}" flags="sx">
+         <xsl:analyze-string select="$stylesheet-string" regex="{$construct-regex}" flags="sx">
             <xsl:matching-substring>
                <xsl:map>
-                  <xsl:for-each select="1 to 10">
+                  <xsl:for-each select="1 to 11">
                      <xsl:map-entry key="." select="regex-group(.)" />
                   </xsl:for-each>
                </xsl:map>
             </xsl:matching-substring>
             <xsl:non-matching-substring>
-               <xsl:message terminate="yes">
-                  <xsl:text expand-text="yes">ERROR: unmatched string: {.}</xsl:text>
-               </xsl:message>
+               <xsl:message terminate="yes" expand-text="yes">ERROR: unmatched string: {.}</xsl:message>
```

Comment:
This would be nice in general, but could you keep the original `xsl:message/xsl:text` structure for the moment?

The reason is that Oxygen XML Editor by default does not include `xsl:message` in **Preferences** - **Editor** - **Format** - **XML** - **Preserve space**. So, after your simplification, applying **Document** - **Source** - **Format and Indent** formats the line as follows
```xslt
               <xsl:message terminate="yes" expand-text="yes">ERROR: unmatched string:
                  {.}</xsl:message>
```
and produces a wrapped message at run time:
```console
ERROR: unmatched string:
                  &lt;
```

Reply:
Done.